src/AttributeTranslator/ArrayAttributeTranslator.php

```diff
@@ -11,6 +11,10 @@
  */
 final class ArrayAttributeTranslator implements AttributeTranslatorInterface
 {
+    /**
+     * @param array $translations Translations array where each key is an attribute name and the corresponding name is
+     * a translation.
+     */
     public function __construct(
         /**
          * @var array<string,string>
```

Comment:
Method `translate()` is left without PHPDoc at all. Using "Ctrl + Q" shows that while some docs are inherited from interface (method description), the others are not (return description). Suggest to add even the copied version or discuss and possibly remove in other places later for consistency.

Reply:
It is IDE bug we need to fill in the issue tracker.

Reply:
https://youtrack.jetbrains.com/issue/WI-70430/phpdoc-is-not-inherited-from-the-interface